src/NServiceBus.Metrics/RawData/RingBuffer.cs

```diff
@@ -16,17 +16,19 @@ class RingBuffer
 
         public struct Entry
         {
-            public Entry(long ticks, long value)
+            public Entry(long ticks, long value, int tag = 0)
```

Comment:
_My reasoning behind this choice. I did not include this in the PR as this sth more like a preference_

We could go with `<T>` but this could encourage to use classes like `string` or others. The flushing mechanism is asynchronous, so my intent was to use the explicit type for now and to discourage to put `object`s in there, which could make them live much longer than needed.